guillotina_amqp/worker.py

```diff
@@ -392,6 +400,44 @@ def cancel(self):
         while len(self._running) > 0:
             await asyncio.sleep(0.01)
 
+    async def check_activity(self, check_after_s=10, noop_after_s=30, kill_after_s=300):
```

Comment:
Can we change `check_after_s` to 30 and `noop_after_s` to 200? That would mean 3 noop requests would be sent before it gets killed.

Otherwise, we'd potentially be sending quite a few of these.

Reply:
sounds good!